Modules/_sqlite/cursor.c

```diff
@@ -610,30 +609,61 @@ _pysqlite_query_execute(pysqlite_Cursor* self, int multiple, PyObject* args)
     }
 }
 
-PyObject* pysqlite_cursor_execute(pysqlite_Cursor* self, PyObject* args)
+/*[clinic input]
+_sqlite3.Cursor.execute as pysqlite_cursor_execute
+
+    sql: unicode
+    parameters: object = NULL
+    /
+
+Executes a SQL statement.
+[clinic start generated code]*/
+
+static PyObject *
+pysqlite_cursor_execute_impl(pysqlite_Cursor *self, PyObject *sql,
+                             PyObject *parameters)
+/*[clinic end generated code: output=d81b4655c7c0bbad input=f146ded55ff8c7d4]*/
 {
-    return _pysqlite_query_execute(self, 0, args);
+    return _pysqlite_query_execute(self, 0, sql, parameters);
 }
 
-PyObject* pysqlite_cursor_executemany(pysqlite_Cursor* self, PyObject* args)
+/*[clinic input]
+_sqlite3.Cursor.executemany as pysqlite_cursor_executemany
+
+    sql: unicode
+    parameters: object = NULL
```

Comment:
Actually the parameter is `seq_of_parameters`, and it is not optional.
```suggestion
    seq_of_parameters: object
```